cmd/mistryd/server.go

```diff
@@ -402,10 +458,18 @@ func (s *Server) HandleServerPush(w http.ResponseWriter, r *http.Request) {
 // non-nil error.
 func (s *Server) ListenAndServe() error {
 	s.Log.Printf("Configuration: %#v", s.cfg)
+	s.setupWorkerPool()
 	go s.br.ListenForClients()
 	return s.srv.ListenAndServe()
 }
 
+func (s *Server) setupWorkerPool() {
```

Comment:
This is too small and I think it should be inlined to `ListenAndServe`. We're not going to use it anywhere else for sure.